distro/pkg/rpm/libnetconf2.spec

```diff
@@ -0,0 +1,64 @@
+Name: libnetconf2
+Version: {{ version }}
+Release: {{ release }}%{?dist}
+Summary: NETCONF protocol library
+Url: https://github.com/CESNET/libnetconf2
+Source: libnetconf2-%{version}.tar.gz
+License: BSD-3-Clause
+
+BuildRequires:  cmake
+BuildRequires:  gcc
+BuildRequires:  make
+BuildRequires:  libyang2-devel
```

Comment:
As you noticed, `libssh-devel` but also `libopenssl-devel` should be added as `BuildRequires`. I am also confused by any `Requires` missing (such as `libyang2`, `libssh`, and `openssl`), that should not be included?

Reply:
I've added the missing `BuildRequires`.

For C/C++ libs, modern `rpm` is able to detect runtime `Requires` quite well so there is no need to specify those explicitly AFAIK.

Here is a relevant portion of `rpm -qp --requires libnetconf2-2.0.1.2.gbe78a4c-1.fc34.x86_64.rpm` output:

```
libcrypt.so.2()(64bit)
libcrypt.so.2(XCRYPT_2.0)(64bit)
libcrypto.so.1.1()(64bit)
libcrypto.so.1.1(OPENSSL_1_1_0)(64bit)
libpthread.so.0()(64bit)
libpthread.so.0(GLIBC_2.2.5)(64bit)
libpthread.so.0(GLIBC_2.3.2)(64bit)
libssh.so.4()(64bit)
libssh.so.4(LIBSSH_4_5_0)(64bit)
libssl.so.1.1()(64bit)
libssl.so.1.1(OPENSSL_1_1_0)(64bit)
libyang.so.2()(64bit)
```

Reply:
Alright, sounds good, did not know that.